cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -872,61 +876,71 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                     expect($("#grading_type").val()).toBe('Lab');
                     expect($("#staff_lock").is(":checked")).toBe(true);
-                    expect($("#id_timed_exam").is(":checked")).toBe(false);
-                    expect($("#id_proctored_exam").is(":checked")).toBe(true);
+                    expect($("#id_timed_exam").is(":checked")).toBe(true);
+                    expect($("#id_proctored_exam").is(":checked")).toBe(false);
                     expect($("#id_not_timed").is(":checked")).toBe(false);
                     expect($("#id_practice_exam").is(":checked")).toBe(false);
                     expect($("#id_time_limit").val()).toBe("02:30");
+                    expect($("#id_hide_exam_after_due_date").is(":checked")).toBe(true);
                 });
 
-                it('can hide the time limit field when the None radio box is selected', function() {
+                it('can hide time limit and hide after due fields when the None radio box is selected', function() {
                     createCourseOutlinePage(this, mockCourseJSON, false);
                     outlinePage.$('.outline-subsection .configure-button').click();
+                    selectAdvancedSettings();
                     setEditModalValues("7/9/2014", "7/10/2014", "Lab", true);
                     selectDisableSpecialExams();
 
                     // id_time_limit_div should be hidden when None is specified
-                    expect($('#id_time_limit_div')).toHaveClass('is-hidden');
+                    expect($('#id_time_limit_div').is(':hidden')).toBe(true);
+                    expect($('.timed-exam-options-list-fields').is(':hidden')).toBe(true);
                 });
 
                 it('can select the practice exam', function() {
                     createCourseOutlinePage(this, mockCourseJSON, false);
                     outlinePage.$('.outline-subsection .configure-button').click();
+                    selectAdvancedSettings();
                     setEditModalValues("7/9/2014", "7/10/2014", "Lab", true);
                     selectPracticeExam("00:30");
                     // id_time_limit_div should not be hidden when practice exam is specified
-                    expect($('#id_time_limit_div')).not.toHaveClass('is-hidden"');
+                    expect($('#id_time_limit_div').is(':visible')).toBe(true);
+                    expect($('.timed-exam-options-list-fields').is(':hidden')).toBe(true);
                     $(".wrapper-modal-window .action-save").click();
                 });
 
                 it('can select the timed exam', function() {
                     createCourseOutlinePage(this, mockCourseJSON, false);
                     outlinePage.$('.outline-subsection .configure-button').click();
+                    selectAdvancedSettings();
                     setEditModalValues("7/9/2014", "7/10/2014", "Lab", true);
-                    selectTimedExam("00:30");
+                    selectTimedExam("00:30", true);
```

Comment:
I do, but not in this test. Here we're only checking to see that the field is visible, so the actual value of the boolean is irrelevant. I'll change this call to not pass any value for clarity.